### PR TITLE
Upgrade everything so that the postgres formula works

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

**Comment by Bjwebb** [comment]
This is needed to install postgres onto the pwyf boxes.

File: requirements.in [hunk 1]
```diff
@@ -1,2 +1,2 @@
-salt-ssh==3002.9
+salt
```
Comment (odscjames) [inline requirements.in:1]: I think it would be good if we still locked, to make clear that upgrading is a major thing?

Comment (Bjwebb) [reply requirements.in:1]: Good idea.

Comment (Bjwebb) [reply requirements.in]: Done

**Comment by Bjwebb** [comment]
@odscjames has tested this against dev servers, so it's ready for merging.

**Review by odscjames — APPROVED** [review]

**Pull request merged**